### DISCUSSION TITLE
Define native element fields as actual input for ACF/SCF fields via "dynamic content" option

### DISCUSSION THREAD

**Discussion by Stml-St** [elementor/elementor#29625] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

My clients are currently forced to fill ACF or SCF fields in the ‘classic’ WordPress editor resp. the Gutenberg Editor and everything else has to be done in the Elementor editor. So there are 2 different places where content has to be maintained for the same page or a post. My feature request would save the user from having to deal with Gutenberg or the Classic Editor. There would only be the Elementor Editor for the user!

### Describe the solution you'd like.

The feature I would like to have is to be able to select an ACF or SCF field under ‘dynamic content’ and then enter the actual content of the field in the Elementor editor field, NOT in Gutenberg or Classic Editor.

### Describe alternatives you've considered.

There is no alternative i could think of. 

### Additional context

I would like to briefly explain this again using an example. For example, if I currently create a custom field with ACF or SCF and a user wants to fill in this field for a specific post, he has to switch to the GutenbergEditor, must scroll down and fill in the field and then switch back to the ElementorEditor to continue working on the content of the post. If it were possible to select the ACF or SCF field in Elementor elements under ‘dynamic content’ and then enter the content for the field directly in the Elementor editor, the user would no longer have to switch to the GutenbergEditor and working on a complex posttype would be much more intuitive than it is at the moment.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by pumadog** [comment]
I would love this!
